src/test/e2e/deck-filtering.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Locator, Page, expect, test } from "@playwright/test";
+import { importDeck, locateCardInSlots } from "./actions";
+import { mockApiCalls } from "./mocks";
+
+test.beforeEach(async ({ page }) => {
+  await mockApiCalls(page);
+});
+
+test.describe("deck filtering", () => {
+  test("filtering works properly", async ({ page }) => {
```

Comment:
So, ran the test with `--repeat-each=45` and indeed was flaky, some 20% failure rate, which was weird, since it was being consistent on my own runs. All failures were on the chromium runner though, andI was testing on ui mode, which must use a different runner.
Anyways, added an extra await after the import from arkhamdb and it seemed to fix it, 45 repeats passed 100%